WcaOnRails/spec/mailers/previews/delegate_status_change_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/wca_id_claim_mailer
```

Comment:
Is this link accurate?

Reply:
And this is a result of the copy-pasting too.